packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -20,15 +20,22 @@ Apply the class `.pt-skeleton` to elements that you would like to cover up with
 dimensions of whatever element the class is applied to. This means that when using skeletons to show loading text, you should use
 some sort of placeholder text that is approximately the length of your expected text.
 
+<div class="pt-callout pt-intent-warning pt-icon-warning-sign">
+  <h5>Usage with focusable elements</h5>
+  When using the `.pt-skeleton` class with focusable elements such as inputs and buttons, be sure to set the `tabindex` attribute
+  to `-1` or the `disabled` attribute to prevent focusing. Failing to do so will allow the browser draw an outline around the
+  skeleton when it is tab-focused.
+</div>
+
 Markup:
 <div class="pt-card">
-  <h5><a class="{{.modifier}}" href="#">My Card Title</a></h5>
+  <h5><a class="{{.modifier}}" href="#" tabindex="-1">My Card Title</a></h5>
   <p class="{{.modifier}}">
     Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget tortor felis. Fusce dapibus metus in dapibus mollis.
     Quisque eget ex diam. Vivamus mattis nisi ut nisl rhoncus convallis. Curabitur id tristique mi, quis elementum dolor. In
     suscipit ut sapien ac tempor. Aenean quis fringilla felis.
   </p>
-  <button type="button" class="pt-button {{.modifier}}">Submit</button>
+  <button type="button" class="pt-button {{.modifier}}" disabled>Submit</button>
```

Comment:
ugh i'm not keen on how this affects the `default` example but not sure what else to do...